nab8balld/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -22,18 +22,18 @@ msgid "Magic 8 Ball"
 msgstr "Gourou"
 
 #: templates/nab8balld/settings.html:10
-msgid "Wake up the guru"
-msgstr "Réveiller le gourou"
```

Comment:
Could you keep the original text ? Thanks!
(Wake up the guru instead of Enable)

Reply:
> Could you keep the original text ? Thanks!

I do have a commit for this (plus other french localization cleanup ).
But how do I push a commit to a PR that is not mine? (never done this before...)

Reply:
ok, I do that.

Reply:
Thanks!

Reply:
I detected a couple of other glitches in french localisation (missing labels).
Can you please pick up from https://github.com/f-laurens/pynab/tree/pr187 the following additional commit:
https://github.com/f-laurens/pynab/commit/a2800c304f372922fb506e232ad69456863bdb63 and push it to this PR?